nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -183,7 +186,7 @@ $(GuidPatchTargetAssemblyReferences)
           BeforeTargets="Link">
 
     <ItemGroup>
-      <CsWinRTGuidPatchedFiles Include="$(MSBuildProjectDirectory)\$(IIDOptimizerInterimDir)*.dll" />
+      <CsWinRTGuidPatchedFiles Include="$(IIDOptimizerInterimDir)*.dll;$(IIDOptimizerInterimDir)*.pdb" />
```

Comment:
Wondering if we should be specific about the dll we copy assuming we know the assembly name.